SConstruct

```diff
@@ -3101,7 +3101,7 @@ if doConfigure :
 		arnoldTestEnv["ENV"]["ARNOLD_PLUGIN_PATH"] = "contrib/IECoreArnold/test/IECoreArnold/plugins"
 		arnoldTest = arnoldTestEnv.Command( "contrib/IECoreArnold/test/IECoreArnold/results.txt", arnoldPythonModule, pythonExecutable + " $TEST_ARNOLD_SCRIPT" )
 		NoCache( arnoldTest )
-		arnoldTestEnv.Depends( arnoldTest, [ arnoldPythonModule + arnoldProceduralForTest + arnoldDriverForTest ] )
+		arnoldTestEnv.Depends( arnoldTest, [ arnoldPythonModule + arnoldProceduralForTest + arnoldDriverForTest + arnoldLibrary ] )
```

Comment:
I'm surprised this is necessary, since arnoldPythonModule already depends on arnoldLibrary.

Reply:
The issue is the cache.  If you make a change to arnoldLibrary that does not produce any changes in the python/IECoreArnold/_IECoreArnold.so binary, then the cache considers the test to have already been run with this exact set of inputs.